contrib/win32/install/install/components/c_server.wxs

```diff
@@ -0,0 +1,138 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Wix
+  xmlns="http://schemas.microsoft.com/wix/2006/wi"
+  xmlns:fw="http://schemas.microsoft.com/wix/FirewallExtension"
+  xmlns:util="http://schemas.microsoft.com/wix/UtilExtension">
+  <Fragment>
+    <ComponentGroup Id="CG_SERVER_FILES" Directory="INSTALLFOLDER">
+      <Component Id="cmp_sftpserver.exe">
+        <File Id="sftpserver.exe" KeyPath="yes" Source="$(var.SOURCEDIR)/sftp-server.exe" Name="sftp-server.exe" />
+      </Component>
+      <Component Id="cmp_sshshellhost.exe">
+        <File Id="sshshellhost.exe" KeyPath="yes" Source="$(var.SOURCEDIR)/ssh-shellhost.exe" Name="ssh-shellhost.exe" />
+      </Component>
+      <Component Id="cmp_opensshevents.man">
+        <File Id="opensshevents.man" KeyPath="yes" Source="$(var.SOURCEDIR)/openssh-events.man" Name="openssh-events.man">
+          <util:EventManifest />
+        </File>
+        <util:XmlFile
+          Id="xmlSetResourceFileName"
```

Comment:
Can you educate me what are we doing here?

Reply:
This modifies the `resourceFileName` to point at the actual installed executable since it's hardcoded in the manifest.